pkg/mappings.bzl

```diff
@@ -337,19 +337,18 @@ pkg_files = rule(
             """,
             default = "",
         ),
-        "strip_prefix": attr.string(
-            doc = """What prefix of a file's path to discard prior to installation.
+        "srcs_strip_prefix": attr.string(
```

Comment:
Do we really have to do this rename?  It will force incompatibility between versions.
That will basically mean I'll have to fork this for Google to keep the old naming for a very long time.